ext/opentelemetry-ext-grpc/src/opentelemetry/ext/grpc/__init__.py

```diff
@@ -16,9 +16,62 @@
 # pylint:disable=import-self
 # pylint:disable=no-name-in-module
 # pylint:disable=relative-beyond-top-level
+from contextlib import contextmanager
 
+import grpc
+from concurrent import futures
 from opentelemetry import trace
 from opentelemetry.ext.grpc.version import __version__
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.auto_instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.ext.grpc.grpcext import intercept_channel, intercept_server
+
+
+class GrpcInstrumentorServer(BaseInstrumentor):
+    trace.set_tracer_provider(TracerProvider())
+    tracer = trace.get_tracer_provider()
```

Comment:
where is this used? either way, we get the tracer provider in the `kwargs` of `_instrument` (if it is not default), we should not be setting our own tracer provider. For example: [psycopg2 instrumentation](https://github.com/open-telemetry/opentelemetry-python/blob/master/ext/opentelemetry-ext-psycopg2/src/opentelemetry/ext/psycopg2/__init__.py#L72)